# Custom message for power meter data
std_msgs/Header header
float32 angular_velocity     # Angular velocity in m/sÂ²
float32 average_power        # Average power in watts
float32 cadence              # Cadence in RPM
float32 instantaneous_power  # Instantaneous power in watts
float32 left_power           # Left power in watts
float32 right_power          # Right power in watts
float32 torque               # Torque in Nm
